kubernetes/istio/istio/src/main/java/org/arquillian/cube/istio/impl/IstioClientAdapter.java

```diff
@@ -59,6 +59,7 @@ public V1alpha3APIGroupDSL v1alpha3() {
         return unwrap().v1alpha3();
     }
 
+    @Deprecated
```

Comment:
Nitpick but for future reference, always add Javadoc with `@deprecated` explanation what to use instead.